pkg/flink/internal/controller/input_controller.go

```diff
@@ -218,17 +222,35 @@ func getUnixBindings() []prompt.Option {
 	return []prompt.Option{
 		prompt.OptionAddASCIICodeBind(
 			prompt.ASCIICodeBind{
-				ASCIICode: []byte{0x1b, 0x62}, // Alt/Option + Arrow Left
+				ASCIICode: []byte{0x1b, 0x62}, // Alt/Option + Arrow Left (ESC + b)
```

Comment:
I can't find any official documentation, but from my experience, typically (although this no longer seems to be the default in terminals) Alt + key being pressed in terminals gets mapped to the input `ESC, key` ( `^[` / `0x1b` is the [`ESC` character](https://en.wikipedia.org/wiki/Escape_character#:~:text=The%20ASCII%20%22escape%22%20character%20(,control%20sequence%20or%20escape%20sequence.)).

Somehow with default terminal settings `Alt + Left` actually results in the same input as my `Alt + b` (`^[b`). Maybe because arrow keys were introduced to the keyboard after `esc` and `b` keys already existed? 🤷 . iterm defaults do a completely weird thing where they completely remap it to another key sequence

Reply:
I see... As a non-expert in that area, I think this is hard to understand from just reading the comment